src/main/java/com/bp3x/raidbot/commands/lfg/RemindCommand.java

```diff
@@ -27,35 +29,51 @@ protected void execute(CommandEvent event) {
 
         String[] args = event.getArgs().split("\\s+");
-        if (args.length == 2) {
+        if (args.length == 2 && (args[1].equalsIgnoreCase("accepted") || args[1].equalsIgnoreCase("tentative"))) {
             // Retrieve event ID data and then add the names of the users in the accepted list
-            Event remindEvent = Event.getEventById(args[1]);
+            Event remindEvent = Event.getEventById(args[0]);
             StringBuilder reminderStringBuilder = new StringBuilder();
 
             if (remindEvent != null) {
-                if (args[0].equalsIgnoreCase("accepted")) {
-                    for (Member player : remindEvent.getAcceptedPlayers()) {
-                        reminderStringBuilder.append("<@!");
-                        reminderStringBuilder.append(player.getUser().getId());
-                        reminderStringBuilder.append(">\n");
-                        reminderStringBuilder.append(LFGConstants.ACCEPTED_EMOJI);
-                        reminderStringBuilder.append("\n");
-                    }
-                } else if (args[0].equalsIgnoreCase("tentative")) {
-                    for (Member player : remindEvent.getTentativePlayers()) {
-                        reminderStringBuilder.append("<@!");
-                        reminderStringBuilder.append(player.getUser().getId());
-                        reminderStringBuilder.append(">");
-                        reminderStringBuilder.append(LFGConstants.TENTATIVE_EMOJI);
-                        reminderStringBuilder.append("\n");
-                    }
+                buildReminderString(remindEvent, event, reminderStringBuilder);
+                log.info("getActiveParticipants = " + remindEvent.getActiveParticipants());
+                if (remindEvent.getActiveParticipants()) {
                     event.getChannel().sendMessage("Players reminded:\n" + reminderStringBuilder).queue();
-                } else {
-                    event.getChannel().sendMessage("Usage: !remind " + remindEvent.getEventId() + " accepted or tentative").queue();
+                }
+                else {
+                    event.getChannel().sendMessage("There are no players to remind.").queue();
                 }
             } else {
                 event.getChannel().sendMessage("That event does not exist.").queue();
             }
         }
+        else {
+            event.getChannel().sendMessage("Usage: `!remind <event id> accepted` or `!remind <event id> tentative`").queue();
+        }
+    }
+
+    private void buildReminderString(Event remindEvent, CommandEvent event, StringBuilder sb) {
+        String[] args = event.getArgs().split("\\s+");
+
+        if (args[1].equalsIgnoreCase("accepted")) {
+            for (Member player : remindEvent.getAcceptedPlayers()) {
+                sb.append("<@!");
+                sb.append(player.getUser().getId());
+                sb.append(">\n");
+                sb.append(LFGConstants.ACCEPTED_EMOJI);
```

Comment:
how does this look w/ multiple players in the list? is it like: 

@playername ✅ 
@playername ✅ 
@playername ✅ 

and 

@playername ❓ 
@playername ❓ 
@playername ❓ 


I am looking in #command-spam in the bot test discord and it looks like the emoji is on a newline after the name, but maybe that is not correct considering you were just testing stuff.